liblangutil/CharStream.h

```diff
@@ -138,6 +144,7 @@ class CharStream
 private:
 	std::string m_source;
 	std::string m_name;
+	bool m_imported_from_AST{false};
```

Comment:
I just realized you used snake-case here instead of camel case. I think we use camel case here (`m_importedFromAST`) (yes the m_ is the exception for member variables)

Reply:
Oh, yes, I sometimes forget that ... I guess old habits die hard lol